code/sonalyze/db/clusterstore.go

```diff
@@ -268,20 +268,25 @@ type parseResult struct {
 // expected case, if caching works well, so worth exploring maybe.
 
 func init() {
-	for i := 0; i < runtime.NumCPU(); i++ {
-		go (func() {
-			uf := NewUstrCache()
-			for {
-				request := <-parseRequests
-				if request.file == nil {
-					return
+	workers := runtime.NumCPU()
+	//workers = 1
+	for i := 0; i < workers; i++ {
+		uf := NewUstrCache()
+		go Forever(
```

Comment:
Arguably something to uplift to main, should have landed with the sonalyzed integration.